### DISCUSSION TITLE
How do I construct or parse a complex type parameter?

### DISCUSSION THREAD

**Discussion by zajanmy** [vfsfitvnm/frida-il2cpp-bridge#540] [Q&A] [answered]
For example, regarding a parameter of type `GoogleMobileAds.Api.RewardedAd`, how can I parse or modify it? Could you share some ideas?
`System.Void RegisterReloadHandler(GoogleMobileAds.Api.RewardedAd ad); `

**Comment by vfsfitvnm** [comment, marked as answer]
I recommend you to take a look at the various snippets in the issues or discussions :smile: there are plenty of examples
